### ISSUE TITLE
ImplicitUsings Are Not Supported in CLI

### ISSUE THREAD

**Issue by warappa** [JosefPihrt/Roslynator#971]
**Product and Version Used**: 
Roslynator.DotNet.Cli: **3.3**
`dotnet --version`: **7.0.100-rc.2.22477.23**
Target framework: **net6.0**

**Steps to Reproduce**:
Create a project with `<ImplicitUsings>enable</ImplicitUsings>`

Add code that uses eg. Linq.

Compile with Visual Studio - works.

Run `roslynator fix project.csproj`

**Actual Behavior**:
Error **CS0518** "The predefined type "System.Void" is not defined or imported." is raised.

**Expected Behavior**:
No error.

**Workaround**:
Add global usings via a `Imports.cs` file manually, like
```csharp
global using System;
global using System.Collections.Generic;
global using System.Threading;
global using System.Threading.Tasks;
global using System.IO;
global using System.Linq;
```

**Comment by JosefPihrt** [comment]
See https://github.com/JosefPihrt/Roslynator/issues/970#issuecomment-1287879985

**Comment by warappa** [comment]
This seems to be fixed.
